CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -30,11 +34,14 @@ public class JavascriptObjectRepository : DisposableResource
     {
         private static long lastId;
         private static readonly object Lock = new object();
+        private const BindingFlags DefaultBindingFlags = BindingFlags.Instance | BindingFlags.Public;
 
         // A hash from assigned object ids to the objects,
         // this is done to speed up finding the object in O(1) time
         // instead of traversing the JavaScriptRootObject tree.
         private readonly Dictionary<long, JavascriptObject> objects = new Dictionary<long, JavascriptObject>();
+        // objects are never removed so we need to try to avoid duplicates, other wise 1000 calls to bound.getSubObject() will create 1000 entries in objects.
+        private readonly Dictionary<object, JavascriptObject> reverseLookup = new Dictionary<object, JavascriptObject>();
```

Comment:
Is there a move descriptive name? `reverseLookup` seems like it's more of a `cache`. Can we make the comment more descriptive? Need to immediately differentiate between the `objects` dictionary, and this one without having to dig through the code to work out what each is for.